tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -143,4 +136,4 @@ if [ "$START_CALDAV" = "1" ]; then
     cd /tracim/backend/
     tracimcli caldav sync -c /etc/tracim/development.ini
 fi
-tail -f /var/log/dpkg.log
+tail -f /var/log/uwsgi/app/tracim_web.log
```

Comment:
Why you change this ?

Uwsgi log is not store in container. We have done this `tail -f /var/log/dpkg.log` to make container running. all necessary log is outside container.

Reply:
> Why you change this ?
> 
> Uwsgi log is not store in container. We have done this `tail -f /var/log/dpkg.log` to make container running. all necessary log is outside container.

I think it makes more sense to block displaying the main applications's log file rather than an unrelated file. When running the container in foreground you directly see the log and it makes it available with `docker logs`.

Reply:
You're right. Maybe its more simple to understand that this log is available in volume if we make `tail -f /var/tracim/logs/tracim_web.log`, what do you think ?

Reply:
> You're right. Maybe its more simple to understand that this log is available in volume if we make `tail -f /var/tracim/logs/tracim_web.log`, what do you think ?

`/var/tracim/` in mounted when you start docker (docker run or docker-compose).

Reply:
You're right, I will change to `/var/tracim/logs/tracim_web.log`.